integration-tests/environments/react-native-test-app/package.json

```diff
@@ -130,7 +131,7 @@
   },
   "dependencies": {
     "base-64": "^1.0.0",
-    "mocha-remote-react-native": "^1.12.2",
+    "mocha-remote-react-native": "^1.12.3",
```

Comment:
This new version avoids transforming errors when the bundle isn't shipped from Metro.